.gitignore

```diff
@@ -3,7 +3,7 @@ target/
 .classpath
 .settings/
 .svn/
-bin/
+/bin/
```

Comment:
???

Reply:
a bin folder is included in the test resources as this is part of the layout below maven home! The intent here is to only exclude compiled classes which will only ever end up in the to-level directory /bin (if at all)!